register_types.cpp

```diff
@@ -70,7 +70,7 @@ void uninitialize_kotlin_jvm_module(ModuleInitializationLevel p_level) {
     if (Engine::get_singleton()->is_project_manager_hint()) { return; }
 #endif
 
-    if (p_level != MODULE_INITIALIZATION_LEVEL_SERVERS) { return; }
+    if (p_level != MODULE_INITIALIZATION_LEVEL_SCENE) { return; }
```

Comment:
Why this has changed ?

Reply:
It was a mistake, the init was set to the scene level but the uninit was set to server. People had bug when closing the game because the destructor of some Godot objects needs singletons to be available, which is not guaranted if you do it later than the Scene level.